<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:/wepik-photo-mode-2022923-115059.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>861</width>
      <height>541</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,
                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);
    border: 2px solid#9708B4;
    border-bottom-color: #9708B4; /* same as the pane color */
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;
    min-width:60ex;
    padding: 2px;
     font: bold, &quot;Comic Sans MS&quot;;
}

QTabBar::tab:selected, QTabBar::tab:hover {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #085394, stop: 0.4 #9708B4,
                                stop: 0.5 #9708B4, stop: 1.0 #9708B4);
}

QTabBar::tab:selected {
    border-color: #9B9B9B;
    border-bottom-color: #9708B4; /* same as pane color */
}
</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>741</width>
        <height>441</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox
{
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string>Ajouter Client</string>
      </property>
      <widget class="QLineEdit" name="LINE_CIN">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}

QDateEdit{
background:transparent;
border:none;
}</string>
       </property>
       <property name="text">
        <string>CIN</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LINE_NOM">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>190</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}

QDateEdit{
background:transparent;
border:none;
}</string>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LINE_PRENOM">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>240</y>
         <width>301</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}

QDateEdit{
background:transparent;
border:none;
}</string>
       </property>
       <property name="text">
        <string>PRENOM</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton_2">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>200</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/icons8-effacer-64.png</normaloff>:/icons8-effacer-64.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
      </widget>
      <widget class="QToolButton" name="B_ajouter">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>110</y>
         <width>81</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="icons/icons.qrc">
         <normaloff>:/icons8-ajouter-64.png</normaloff>:/icons8-ajouter-64.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Modifier/Supprimer</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>421</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LINE_C">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>80</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}

QDateEdit{
background:transparent;
border:none;
}</string>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LINE_N">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>130</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}

QDateEdit{
background:transparent;
border:none;
}</string>
      </property>
      <property name="text">
       <string>NOM</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="LINE_P">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}

QDateEdit{
background:transparent;
border:none;
}</string>
      </property>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </widget>
     <widget class="QToolButton" name="pb_supp">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>250</y>
        <width>61</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons/icons.qrc">
        <normaloff>:/icons8-effacer-64 (1).png</normaloff>:/icons8-effacer-64 (1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="pb_modif">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>250</y>
        <width>71</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons/icons.qrc">
        <normaloff>:/icons8-engrenage-64.png</normaloff>:/icons8-engrenage-64.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tab_client">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>90</y>
        <width>641</width>
        <height>361</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_11">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>50</y>
        <width>481</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}

QDateEdit{
background:transparent;
border:none;
}</string>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_5">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>40</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons/icons.qrc">
        <normaloff>:/312429297_445290024180468_1769385482268134196_n.png</normaloff>:/312429297_445290024180468_1769385482268134196_n.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
     <widget class="QToolButton" name="toolButton_6">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>40</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
background:#01166B;
border-radius:60px;
}
QToolButton{
background:#01166B;
border-radius:60px;} 

QToolButton::hover{
background:#9708B4;
border-radius:60px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="icons/icons.qrc">
        <normaloff>:/312551831_410074381158418_5360117734813686178_n - Copie - Copie.png</normaloff>:/312551831_410074381158418_5360117734813686178_n - Copie - Copie.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Reclamation </string>
     </attribute>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>491</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="icons/icons.qrc">:/icons8-carnet-d'adresses-64.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>240</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton{
background:#9708B4;
border-radius:60px;
}


QPushButton::hover{
background:rgb(170, 85, 255);
border-radius:60px;
}

</string>
      </property>
      <property name="text">
       <string>valider</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>statistique</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>381</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox
{
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string>Stat</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>391</width>
        <height>421</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox
{color:rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string>Fidelité</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1189</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
